client/src/components/RichTextEditor.js

```diff
@@ -149,13 +148,6 @@ const serializeDebounced = debounce((node, onChange) => {
 }, 100)
 
 const deserialize = element => {
-  if (
-    element.nodeName === "#text" &&
-    _isEmpty(_trim(element.textContent, "\n"))
-  ) {
-    // Skip empty elements
-    return null
-  }
```

Comment:
Please add a testcase, e.g. add a dummy report to the demo data with this empty paragraph in it, and navigate to that report in a client-side test. Then also reverse the order of the commits (so first add the test, which should fail, then remove this code, after which the test should pass).

Reply:
Test cases covered in PR #4089